app/client/app/components/attendance-list/attendance-list.component.ts

```diff
@@ -274,11 +274,20 @@ export class AttendanceListComponent implements OnInit {
   }
 
   displayErrorAlert(error) {
-    swal(
-      error.title,
-      error.msg,
-      'error'
-    );
+    if (error.title === "Auth Error") {
+      this.router.navigate(['/login']);
+      swal(
+        error.title,
+        error.msg,
+        'info'
+      );
+    } else {
+      swal(
+        error.title,
+        error.msg,
+        'error'
+      );
+    }
```

Comment:
This `displayErrorAlert` method looks identical to `displayErrorAlert` methods in `attendance-report` and `case-notes`. Could this code be shared through a common inherited method or some other means?

Reply:
Absolutely. Will have that cleaned up for next pull request.